### ISSUE TITLE
Resolve renv reproducibility issue

### ISSUE THREAD

**Issue by sapolikanov** [sapolikanov/voting-abroad#2]

**Comment by sapolikanov** [comment]
I think I have cleared up most of the issues surrounding the un-reproducibility of data. .gitignore seems to work good, and lack of misc files reduces interference with renv. There is an issue with the modelsummary package, one of its (full) dependencies, Icens, has been removed from CRAN and is only available via the Bioconductor. I do not plan to implement code to download it from there.

I have changed the check_packages.R file to install "most" dependencies (see renv definition of the concept). This seems to work. 

However, I cannot fully confirm or reject that the project is reproducible following [release 1.0.1](https://github.com/sapolikanov/voting-abroad/releases/tag/v1.0.1) due to high computational intensity of the models. Since I cannot run it at reasonable time, neither can anyone trying to replicate analyses. 

Therefore I have opened #7 to deal with that first. I do not close this issue yet.
